### PR TITLE
Add sole tenant resources to janitor cleanup list

### PR DESCRIPTION
**Pull request by jlowdermilk**

### CODE REVIEW

**Comment by krzyzacy** [comment]
assume it worked locally for you? :-)

**Comment by jlowdermilk** [comment]
> assume it worked locally for you? :-)
It does work when run locally, but appears to require the zone/region to be set as a gcloud property. Is that expected, or does that mean I should put zone/region respectively as the Resource condition?

File: boskos/janitor/janitor.py [hunk 1]
```diff
@@ -49,6 +49,8 @@
     Resource('', 'compute', 'instance-groups', None, 'zone', 'Yes', False, True),
     Resource('', 'compute', 'instance-groups', None, 'zone', 'No', False, True),
     Resource('', 'compute', 'instance-templates', None, None, None, False, True),
+    Resource('', 'compute', 'sole-tenancy', 'node-groups', None, None, False, True),
+    Resource('', 'compute', 'sole-tenancy', 'node-templates', None, None, False, True),
```
Comment (krzyzacy) [inline boskos/janitor/janitor.py]: the 5th arg has a 'zone' or 'region' condition which can be passed into the command

Comment (jlowdermilk) [reply boskos/janitor/janitor.py]: Yep that fixed it. I misunderstood the purpose of the arg.

**Comment by jlowdermilk** [comment]
local run passing now

**Comment by krzyzacy** [comment]
/lgtm

will build a new image

**Comment by krzyzacy** [comment]
is there a gcloud version requirement? Currently I believe it's pretty old.. (let me double check)

**Comment by krzyzacy** [comment]
```
senlu@senlu:~/work/src/k8s.io/test-infra$ docker run --entrypoint=/bin/sh -it gcr.io/k8s-testimages/janitor:v20190215-1a8f5436c
# gcloud version
Google Cloud SDK 209.0.0
alpha 2018.07.16
beta 2018.07.16
bq 2.0.34
core 2018.07.16
gsutil 4.33
kubectl 


Updates are available for some Cloud SDK components.  To install them,
please run:
  $ gcloud components update

# gcloud compute sole-tenancy node-groups list
ERROR: (gcloud.compute) Invalid choice: 'sole-tenancy'.
This command is available in one or more alternate release tracks.  Try:
  gcloud beta compute sole-tenancy
```

hummm let me try to bump that

**Pull request merged**